src/Jackett.Common/Indexers/TorrentHeaven.cs

```diff
@@ -95,24 +78,17 @@ private new ConfigurationDataCaptchaLogin configData
             AddCategoryMapping(71, TorznabCatType.PCMac, "APPLICATIONS/Mac");
         }
 
-        public override async Task<ConfigurationData> GetConfigurationForSetup()
+        private new ConfigurationDataCaptchaLogin configData => (ConfigurationDataCaptchaLogin)base.configData;
+
+        private string IndexUrl => SiteLink + "index.php";
+
+        public override string[] LegacySiteLinks { get; protected set; } =
```

Comment:
Why are you moving these attributes after the constructor?
Fields should be before constructor => https://stackoverflow.com/a/310967

Reply:
The default ReSharper setting moves them that way, so when I run the code cleanup (Ctrl+E, C) this is the result.
![image](https://user-images.githubusercontent.com/2086652/78460330-827a4300-7685-11ea-8e2e-07b23e243af1.png)